libs/sdk-ui/src/base/results/internal/dataAccessDigest.ts

```diff
@@ -251,7 +251,7 @@ function createDataSeriesDigest(
     const dimIdx = locations[0];
     const headerItems = dataViewHeaders(dataView, dimIdx);
     const measureHeaders =
-        headerItems.find((headers): headers is IResultMeasureHeader[] => isResultMeasureHeader(headers[0])) ??
+        headerItems.find((headers): headers is IResultMeasureHeader[] => isResultMeasureHeader(headers[0]))?.filter(isResultMeasureHeader) ??
```

Comment:
we need to fix this somehow. If having this way, it prevents column totals from being added in the standard (metrics in cols) case.